build.gradle

```diff
@@ -47,27 +53,27 @@ dependencies {
     google()
   }
 
+  // update
   compileOnly 'com.segment.analytics.android:analytics:4.10.0'
```

Comment:
let's update analytics to its latest version. `4.10.0` will have google privacy issue